src/ast/mod.rs

```diff
@@ -298,6 +305,20 @@ impl fmt::Display for Expr {
             }
             Expr::Exists(s) => write!(f, "EXISTS ({})", s),
             Expr::Subquery(s) => write!(f, "({})", s),
+            Expr::Parameter {
```

Comment:
Why this is a variant of `Expr`?

Reply:
I will move this out, and add some tests